integr8ly/integreatly-operator
# Bumps OO to v3.0.8 and adds bearer token to blackbox exporter

# Issue link
JIRA: https://issues.redhat.com/browse/MGDAPI-2282

# What
This fix adds a bearer token to the `black-box-config` ConfigMap in the `redhat-rhoam-observability` namespace. The bearer token belongs to a new ServiceAcccount `blackbox-exporter` which is granted `get` permissions to `namespaces`. This allows the `probe_success` query that is emitted by the Prometheus Probes to be authenticated by the proxy that is in front of the `grafana-ui` service. This fixes the bug in the `Endpoints Detailed` grafana dashboard that incorrectly showed the `grafana-ui` service as `DOWN`. 

This fix also includes bumping the `observability-operator` version to v3.0.8 in order to pull in the new feature that was added to the observability-operator repo in this [PR](https://github.com/redhat-developer/observability-operator/pull/94).

# Verification steps
- Provision an OSD cluster at https://qaprodauth.cloud.redhat.com/openshift/
- Checkout this PR
- Install RHOAM onto the OSD cluster by running the RHOAM operator locally:
```
  export USE_CLUSTER_STORAGE="true"
  export INSTALLATION_TYPE=managed-api
  INSTALLATION_TYPE=managed-api make cluster/prepare/local
  patch_in_prow &
  INSTALLATION_TYPE=managed-api make code/run
```
- Wait for the installation to Complete:
```
  oc get rhmi rhoam -n redhat-rhoam-operator -o json | jq .status.stage
```
- Navigate to the `grafana-route` in the `redhat-rhoam-observability` namespace
- Open the `Endpoints Detailed` dashboard in the `redhat-rhoam-observability` folder
- Scroll down to the `grafana-ui UP/DOWN Status` row and confirm that the `grafana-ui` panel is green and displaying `UP`

![grafana-ui-status](https://user-images.githubusercontent.com/80267718/145595440-c5cd425a-372e-4872-a3ba-744542ea7317.png)
